go.mod

```diff
@@ -216,3 +216,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	pgregory.net/rapid v0.4.0 // indirect
 )
+
+replace github.com/karrick/godirwalk => github.com/N-o-Z/godirwalk v1.17.3
```

Comment:
1. Suggest to move this fork into treeverse org and not keep it on a specific user
2. Alt. remove the use of this package if it is not too complex - can be in a different PR